website/content/en/docs/core-tasks/bumping-sdk-version/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Operator SDK Version migration
```

Comment:
It is inside of the "core task" which I also think that needs to be renamed. 
1. However, IMO it should be outside on top of Contribution Guidelines. WDYT? 

Also, find a better name to illustrate its purpose would be nice as well. 
2. I'd suggest replacing `Operator SDK Version migration` for `Upgrade SDK Version`. See;

<img width="257" alt="Screenshot 2020-07-10 at 20 59 55" src="https://user-images.githubusercontent.com/7708031/87193321-60313a80-c2f0-11ea-81d0-a6c3d65b5827.png">

3. Do we really need keep these `Migration` doc from  v0.0.x to v0.1.0?

Reply:
@camilamacedo86 at first glance of the docs it feels like the ordering should be more like: 

- Operator SDK Version upgrade guide
- v0.0.x to v0.1.0
- v0.0.x to v0.18.0
- v0.0.x to v0.19.0

I'm updating the PR, please let me know if I misread the docs/it needs more reordering.